### ISSUE TITLE
TensorBoard not accessible via browser

### ISSUE THREAD

**Issue by mandar2812** [eaplatanios/tensorflow_scala#58]
When ever I run any of the examples I get a log message that tensorboard has been started at [host]:[port] but I am never able to actually access it in my browser, I get `Failed to load resource: An SSL error has occurred and a secure connection to the server cannot be made.`

**Comment by eaplatanios** [comment]
I’ll look into this when I get home, but could you try running it locally and make sure that port 6006 is accessible?

**Comment by mandar2812** [comment]
So if i run `tensorboard --logdir tmp/cifar_summaries/` separately from my python 3.5 virtual env, tensorboard runs fine on `0.0.0.0:6006`. But trying the same when DynaML is executing [`cifar.sc`](https://github.com/transcendent-ai-labs/DynaML/blob/master/scripts/cifar.sc) does not work.

```
2017-11-24 13:05:43.338 [main] INFO  CIFAR Data Loader - Finished loading the CIFAR dataset.
Building the logistic regression model.
Training the linear regression model.
2017-11-24 13:05:44.130 [main] INFO  Learn / Hooks / TensorBoard - Launching TensorBoard in '0.0.0.0:6006' for log directory '/Users/mandar/tmp/cifar_summaries'.
2017-11-24 13:05:44.515 [main] INFO  Variables / Saver - Restoring parameters from '/Users/mandar/tmp/cifar_summaries/model.ckpt-5505'.
2017-11-24 13:05:51.640 [main] INFO  Learn / Hooks / Checkpoint Saver - Saving checkpoint for step 5506.
2017-11-24 13:05:51.641 [main] INFO  Variables / Saver - Saving parameters to '/Users/mandar/tmp/cifar_summaries/model.ckpt'.
2017-11-24 13:06:20.111 [main] INFO  Learn / Hooks / Checkpoint Saver - Saving checkpoint for step 5606.
2017-11-24 13:06:20.111 [main] INFO  Variables / Saver - Saving parameters to '/Users/mandar/tmp/cifar_summaries/model.ckpt'.
2017-11-24 13:06:46.311 [main] INFO  Learn / Hooks / Checkpoint Saver - Saving checkpoint for step 5706.
2017-11-24 13:06:46.311 [main] INFO  Variables / Saver - Saving parameters to '/Users/mandar/tmp/cifar_summaries/model.ckpt'.
2017-11-24 13:07:11.581 [main] INFO  Learn / Hooks / Checkpoint Saver - Saving checkpoint for step 5806.
2017-11-24 13:07:11.581 [main] INFO  Variables / Saver - Saving parameters to '/Users/mandar/tmp/cifar_summaries/model.ckpt'.
2017-11-24 13:07:36.424 [main] INFO  Learn / Hooks / Checkpoint Saver - Saving checkpoint for step 5906.
2017-11-24 13:07:36.424 [main] INFO  Variables / Saver - Saving parameters to '/Users/mandar/tmp/cifar_summaries/model.ckpt'.
2017-11-24 13:08:00.379 [main] INFO  Learn / Hooks / Termination - Stop requested: Exceeded maximum number of steps.
2017-11-24 13:08:00.380 [main] INFO  Learn / Hooks / Checkpoint Saver - Saving checkpoint for step 6006.
2017-11-24 13:08:00.380 [main] INFO  Variables / Saver - Saving parameters to '/Users/mandar/tmp/cifar_summaries/model.ckpt'.
2017-11-24 13:08:00.524 [main] INFO  Learn / Hooks / Termination - Stop requested: Exceeded maximum number of steps.
Train accuracy = 0.49962F
Test accuracy = 0.4247F

```

**Comment by sbrunk** [comment]
In #60 I've changed the call to tensorboard so it should work with Python 3 as well (and prints a warning if it can't start tensorboard). It works for me if I run Scala within the virtual env.

Perhaps this works for you too.

**Comment by eaplatanios** [comment]
@mandar2812 @sbrunk I'll look into the PR and update this.

**Comment by eaplatanios** [comment]
@mandar2812 This must have been fixed in #60. Please re-open this issue if there's still a problem and thanks @sbrunk for the fix. :)
